roosevelt.js

```diff
@@ -8,6 +8,7 @@ const path = require('path')
 const os = require('os')
 const fs = require('fs')
 const fsr = require('./lib/tools/fsr')()
+const reload = require('reload')
```

Comment:
We want to ensure that all dev dependencies are isolated behind dev mode flags in such a way that one could delete all dev dependencies and still operate a roosevelt app without errors. To that effect all references to reload and its functionality should ideally be isolated into their own file if possible, or at the very least buried under a development mode check.

Reply:
Now hides all calls to dev dependencies behind dev mode flags